contrib/freealut/template.py

```diff
@@ -0,0 +1,19 @@
+pkgname = "freealut"
+pkgver = "1.1.0"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = ["cmake", "ninja", "pkgconf"]
+makedepends = ["openal-soft-devel"]
+pkgdesc = "OpenAL Utility Toolkit (ALUT)"
+maintainer = "ogromny <ogromnycoding@gmail.com>"
+license = "GPL-2.0-or-later"
+url = "https://github.com/vancegroup/freealut"
+source = f"{url}/archive/refs/tags/freealut_{pkgver.replace('.', '_')}.tar.gz"
+sha256 = "c3880d7dbb90c7db54e1d6b88aa0a34dd9e2d828fd389be0d2cbb2632b0885dd"
+# freealut has no tests
```

Comment:
there are, they're just manually ran from the build dir. same for toxic below apparently

you could see if they make sense to run

Reply:
I don't see the point of running the test of freealut, `toxic`'s tests are using docker, and if toxic compiles it should just run fine, so I don't think the tests are mandatory for these two, what do you think ?

Reply:
sure